htdocs/lib2/OcSmarty.class.php

```diff
@@ -13,39 +13,24 @@
 
 class OcSmarty extends Smarty
 {
-
     public $name = 'sys_nothing';
-
     public $main_template = 'sys_main';
-
     public $bench = null;
-
     public $compile_id = null;
-
     public $cache_id = null;    // This is a smarty caching ID, not a caches.cache_id.
-
     public $title = '';
-
     public $menuitem = null;
-
     public $nowpsearch = false;
-
     public $change_country_inpage = false;
-
-    // no header, menu or footer
+	// no header, menu or footer
     public $popup = false;
```

Comment:
Oops - in the original diff the comment was moved to the left, here it is shown moved to the right ?!